Readme.md

```diff
@@ -31,9 +36,4 @@ function onFocusedCellChanging(s, e) {
 }
 ```
 
-
-<p><strong>Note</strong>: the FocusedCellChanging event can't prevent editor focusing if the <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebASPxGridView_SettingsEditingtopic">ASPxGridView.SettingsEditing</a> -> <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebASPxGridViewEditingSettings_BatchEditSettingstopic">BatchEditSettings</a> -> <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebGridViewBatchEditSettings_EditModetopic">EditMode</a> property is set to Row.</p>
-
-<br/>
-
-
+This technique is not applicable if you set the [`EditMode`](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewBatchEditSettings.EditMode) property to **Row**. A user can focus and edit any cell in the row switched to edit mode except for cells in read-only columns (the [`GridViewDataColumn.ReadOnly`](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewDataColumn.ReadOnly) property).
```

Comment:
```suggestion
This technique is not applicable if you set the [`EditMode`](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewBatchEditSettings.EditMode) property to **Row**. A user can focus and edit any cell in rows switched to edit mode except for cells in read-only columns (the [`GridViewDataColumn.ReadOnly`](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewDataColumn.ReadOnly) property).
```

Reply:
As a result of the discussion, we've decided to keep the "the row".